### PR TITLE
First ObjVisSAP in latex

### PR DESCRIPTION
**Pull request by jesusjuansalgado**
First ObjVisSAP in latex

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: Check the IVOA document
+
+env:
+  doc_name:
```
Comment (molinaro-m) [inline .github/workflows/build.yml]: Here the document shortname is missing, ObjVisSAP, to let the build work

File: .github/workflows/preview.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+name: Update PDF Preview
+
+env:
+  doc_name:
```
Comment (molinaro-m) [inline .github/workflows/preview.yml]: Same here: missing the ObjVisSAP value for doc_name.

**Review by molinaro-m — COMMENTED** [review]
The document builds and seems fine.
I'd prefer the two doc_name in the CI .yml files to be set, to allow the continuous integration work properly.

File: .github/workflows/build.yml (hunk 1 above)
Comment (jesusjuansalgado) [reply .github/workflows/build.yml]: Done

File: .github/workflows/preview.yml (hunk 2 above)
Comment (jesusjuansalgado) [reply .github/workflows/preview.yml]: Done

**Review by molinaro-m — APPROVED** [review]
With the doc_name in the CI configurations, I approve this PR to have the doc ready.

**Pull request merged**